docs/delegators/delegator-faq.md

```diff
@@ -1,6 +1,6 @@
----
```

Comment:
Any updates on having this not rendered in markdown?

Reply:
`<!---` i believe solves this we are doing it in most places, if we aren't then will change it